migrations/sysdb/00001-collections.sqlite.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE collections (
+    id TEXT PRIMARY KEY,
+    name TEXT NOT NULL,
+    topic TEXT NOT NULL
+);
+
+CREATE TABLE collection_metadata (
```

Comment:
Right now our code doesn't assume collection_metadata is flat. This does - probably worth a discussion.

Reply:
Also this makes migrating to supporting nested metadata in the future a little more complicated.